source/module_md/fire.h

```diff
@@ -41,7 +41,7 @@ class FIRE : public MD_base
      */
     void check_fire(void);
 
-    double max;         ///< max force
+    double max=0.0;         ///< max force
```

Comment:
put in the constructor

Reply:
The variable has been putted in the constructor